src/m365/pp/commands/environment/environment-get.spec.ts

```diff
@@ -11,9 +11,39 @@ const command: Command = require('./environment-get');
 
 describe(commands.ENVIRONMENT_GET, () => {
   const name = '5ca1c616-6060-46ba-abc1-18d312f1cb3a';
+  const nameEnvironmentResponse: any = {
+    "value": [
+      {
+        "id": `/providers/Microsoft.BusinessAppPlatform/scopes/admin/environments/${name}`,
+        "type": "Microsoft.BusinessAppPlatform/scopes/environments",
+        "location": "unitedstates",
+        "name": `${name}`,
+        "properties": {
+          "displayName": "My Power Platform Environment",
+          "isDefault": false
+        }
+      }
+    ]
+  };
+
+  const defaultEnvironmentResponse: any = {
```

Comment:
Right now you have 2 response objects `nameEnvironmentResponse` and `defaultEnvironmentResponse`. Could we merge them together to 1 response?
```
value: [
{ // Environment 1 },
{ // Environment 2 }
]
```

Let's use this response in all tests so we actually test that we do our filtering right.